lua/wire/client/text_editor/texteditor.lua

```diff
@@ -2325,6 +2357,16 @@ function EDITOR:AC_Check( notimer )
 			self:AC_SetVisible( false )
 			return
 		end
+		
+		if suggestions[1].str( suggestions[1] ) == "vec" or suggestions[1].str( suggestions[1] ) == "vec4" then
+			for k=1, table.Count(PickColorFuncs) do
```

Comment:
I did use `#t` but it gave me an error, somehow it was detecting it as a string indexed table, I'll try with the method you said.